CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 The noteworthy changes for each ExMachina version are included here. For a
 complete changelog, see the git history for each version via the version links.
 
+## [0.5.0] - November 18, 2015
+
+### Changed
+
+- Factories were simplified so that `attrs` is no longer required. See [70a0481] and [issue #56]
+- ExMachina.Ecto.assoc/3 was removed. You can now use build(:factory) instead. See discussion in [issue #56]
```

Comment:
It's not on this branch, but https://github.com/thoughtbot/ex_machina/commit/7c6704706cffa7285a608049a1b1f10784790fdd was also added/fixed since 0.4.0